packages/flutter/lib/src/gestures/team.dart

```diff
@@ -114,11 +114,12 @@ class _CombiningGestureArenaMember extends GestureArenaMember {
 /// [AndroidView] uses a team with a captain to decide which gestures are
 /// forwarded to the native view. For example if we want to forward taps and
 /// vertical scrolls to a native Android view, [TapGestureRecognizer]s and
-/// [VerticalDragGestureRecognizer] are added to a team with a captain(the captain is set to be a
-/// gesture recognizer that never explicitly claims the gesture).
-/// The captain allows [AndroidView] to know when any gestures in the team has been
-/// recognized (or all other arena members are out), once the captain wins the
-/// gesture is forwarded to the Android view.
+/// [VerticalDragGestureRecognizer] are added to a team with a captain (the
+/// captain is set to be a gesture recognizer that never explicitly claims the
+/// gesture).
+/// The captain allows [AndroidView] to know when any gestures in the team has
```

Comment:
if you intend this to be a new paragraph in the API docs you'll need a blank line here too, fwiw.

Reply:
Thanks - from what I see this was not a new paragraph previously either. Although that does not make sense from a Markdown perspective because the line will just be wrapped after the other.

I will add two trailing spaces for now (which will put the "The captain ..." sentence into a new line without a blank line in between).